src/components/RepoSearchForm.tsx

```diff
@@ -11,33 +11,53 @@ import Search from 'antd/lib/input/Search';
 export const GithubSearchRepositories: React.FC = () => {
     const dispatch: any = useDispatch();
     const [searchName, setSearchName] = useState('');
-    const { list, isLoading } = useSelector((state: StoreState) => state.gitHubRepositories);
-
+    const { list, isLoading, total_repositories_count } = useSelector((state: StoreState) => state.gitHubRepositories);
```

Comment:
Use `camelCase` for variable names but not `snake_case`.
Rename `{ ..., total_repositories_count: totalRepositoriesCount }`